src/MainWindow.vala

```diff
@@ -59,18 +67,34 @@ public class Maya.MainWindow : Hdy.ApplicationWindow {
         weak Gtk.IconTheme default_theme = Gtk.IconTheme.get_default ();
         default_theme.add_resource_path ("/io/elementary/calendar");
 
-        var headerbar = new Calendar.Widgets.HeaderBar ();
+        headerbar = new Calendar.Widgets.HeaderBar ();
+
+        error_label = new Gtk.Label (null);
+        error_label.show ();
+
+        error_bar = new Gtk.InfoBar () {
+            message_type = Gtk.MessageType.ERROR,
+            revealed = false,
+            show_close_button = true
+        };
+        error_bar.get_content_area ().add (error_label);
 
-        var infobar_label = new Gtk.Label (null);
-        infobar_label.show ();
+        string title = _("Network Not Available.");
+        string details = _("While offline, a local copy of previously added calendars will be displayed.");
```

Comment:
I think it's obvious that a cached copy is displayed. This is more similar to how we handle it in AppCenter, where we explain what connecting will let you do:

```suggestion
        string details = _("Connect to the Internet to see additional details and new events from online calendars.");
```

Reply:
Thank you! 😊